pkg/operator/staticpod/controllers.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/openshift/library-go/pkg/operator/unsupportedconfigoverridescontroller"
 	"github.com/openshift/library-go/pkg/operator/v1helpers"
 
+	"github.com/openshift/client-go/config/informers/externalversions"
```

Comment:
nit: group imports, move up to line `26`

Reply:
Done